packages/client/src/components/OrderDataCompletion/OrderDataCompletion.tsx

```diff
@@ -155,9 +227,12 @@ const OrderDataCompletion = () => {
                   <RadioGroup
                     aria-labelledby='demo-radio-buttons-group-label'
                     name='radio-buttons-group'
-                    onChange={(e) => fillAddressForm(e.target.value)}
+                    onChange={(e) => {
+                      fillAddressForm(e.target.value);
+                      setAddressId(JSON.parse(e.target.value).id);
+                    }}
                   >
-                    {addresses.map((e) => (
+                    {address.map((e: AddressDto) => (
```

Comment:
Nie trzeba (ale można :+1:) jawnie podawać typu, bo TS inferuje.

Reply:
coś jest nie tak, bo ja to już dodałem dawno, nie wiem czemu widzisz bez tej zmiany, bo ja na gicie widzę, że jest to dodane